scripts/class/Hero.js

```diff
@@ -5,6 +5,20 @@ class Hero {
     */
     constructor(level){
         //TODO generate initial hero/random monster with random stats
+        this.name = '';
```

Comment:
Where does this name come from? The constructor should be able to provide a random name, or at least user specified name.